static/js/stream_events.js

```diff
@@ -99,6 +99,9 @@ exports.mark_subscribed = function (sub, subscribers, color) {
         subs.update_settings_for_subscribed(sub);
     }
 
+    // update navbar if necessary
+    tab_bar.maybe_rerender_title_area_for_stream(sub);
```

Comment:
Why are these this function rather than `update_subscribers_ui`?

Reply:
Updated the commit description with this:
```
It would have been better to just have a single call within
server_events_dispatch but it seems difficult due to the way of
mark_subscribed and mark_unsubscribed are structured.

stream_events.mark_unsubscribed conditionally calls
subs.update_settings_for_unsubscribed which calls
subs.rerender_subscriptions_settings and as such handles the update
for the subscriptions modal on its own. Hence, we simply rely on the
stream_data.update_calculated_fields to ensure the subscriber counts
are updated and make a call to
tab_bar.maybe_rerender_title_area_for_stream(sub).

stream_events.mark_subscribed is similar.
```

Is it alright?